test/uk/gov/ons/sbr/models/patch/JsonPatchOperationSpec.scala

```diff
@@ -0,0 +1,83 @@
+package uk.gov.ons.sbr.models.patch
+
+import org.scalatest.{ FreeSpec, Matchers }
+import play.api.libs.json._
+import uk.gov.ons.sbr.models.patch.OperationTypes.{ Add, Remove, Replace, Test }
+
+class JsonPatchOperationSpec extends FreeSpec with Matchers {
+
+  private trait Fixture {
+    def readAsJsonPatchOperation(jsonStr: String): JsResult[JsonPatchOperation] =
+      Json.parse(jsonStr).validate[JsonPatchOperation]
+  }
+
+  "A JsonPatchOperation" - {
+    "can be read from Json" - {
+      "when an add operation" in new Fixture {
+        val jsonStr = """{"op": "add", "path": "/a/b/c", "value": ["foo", "bar"]}"""
```

Comment:
Do we ever use multiple values? `"value": ["foo", "bar"]`

Reply:
We do not have a use-case for this in the context of editing unit links.

However we do support it, as value has type ```JsValue```.
The ```JsonPatchOperation``` class is concerned solely with representing the Json patch specification, and these examples are actually taken directly from RFC6902.

In our application it would be the responsibility of our services to reject such a value for unit-links not ```JsonPatchOperation```.